BE/OwnershipAndControl/Executives.rdf

```diff
@@ -256,171 +307,93 @@
-		</rdfs:subClassOf>
+		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;Signatory"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;OrganizationMember"/>
 		<rdfs:label>corporate officer</rdfs:label>
-		<skos:definition>The executives of a corporation charged with certain operational responsibilities. Typically appointed by the board of directors, the corporate officers usually include the Chief Executive Officer (CEO), Chief Financial Officer (CFO), President, and in some corporations the Chief Operating Officer.</skos:definition>
-		<skos:editorialNote>Some party which is recognized by law as having the role and responsibilities defined for a Public Officer as described in that law.</skos:editorialNote>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.investorwords.com/15091/corporate_officers.html</fibo-fnd-utl-av:definitionOrigin>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-be-oac-exec;CorporateOfficerCapacity">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-law-lcap;DelegatedLegalAuthority"/>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isConferredBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-be-oac-exec;CompanyLaw"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:label>corporate officer capacity</rdfs:label>
-		<skos:definition>the capacity vested in a public officer of a company</skos:definition>
-		<skos:editorialNote>This is framed as a capacity: -an individual acts in their capacity of the role as CFO, CEO etc. but their legal liability rests in their relationship with the board itself i.e. comes from their membership on the board from a corporate standpoint.</skos:editorialNote>
-		<skos:scopeNote>Originally framed with reference to incorporated company, this term and its related terms have been re-framed as applying to any Body Corporate. May be specialized for Incorporated Companies and other Bodies Corporate at some future point.</skos:scopeNote>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-be-oac-exec;CorporateOfficerDuty">
-		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;CompanyStatutoryObligation"/>
-		<rdfs:label>corporate officer duty</rdfs:label>
-		<skos:definition>some duty incumbent upon some officer of a company</skos:definition>
+		<skos:definition>high-level management executive of a corporation or other organization, hired by the board of directors or the business owner(s), charged with certain operational responsibilities</skos:definition>
+		<skos:example>Corporate officers may include a Chief Executive Officer (CEO), Chief Financial Officer (CFO), president, vice president(s), and in some cases a Chief Operating Officer (COO), Chief Compliance Officer (CCO), or other executive responsible for a critical function in the organization.</skos:example>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-be-oac-exec;Executive">
 		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;LegallyDelegatedAuthority"/>
```

Comment:
Surely they HAVE authority not ARE authority.
Is there no realtionship with Corporate Officer - are they orthogonal?

Reply:
So the way legally delegated authority is defined, that's the person or organization that is the authority, and there is a restriction saying that they have (possess) delegated legal authority.  I know it's interesting language, but conceptually what's there is correct.  If you can think of a better name after reviewing this in Protege or another tool rather than in the text of the ontology, after running a reasoner, it will make more sense.